source/OptimaJet.DWKit.StarterApplication/Models/Organization.cs

```diff
@@ -11,13 +11,13 @@ public class Organization : Identifiable
         [Attr("name")]
         public string Name { get; set; }
 
-        [Attr("websiteUrl")]
+        [Attr("website-url")]
         public string WebsiteUrl { get; set; }
 
-        [Attr("buildEngineUrl")]
+        [Attr("build-engine-url")]
         public string BuildEngineUrl { get; set; }
 
-        [Attr("buildEngineApiAccessToken")]
+        [Attr("build-engine-api-access-token")]
```

Comment:
see below: should organizationMemberships be changed to organization-memberships like in user?

Reply:
yeah, all the jsonapi stuff should be hyphenated